sql/internal/specutil/types.go

```diff
@@ -38,11 +39,25 @@ func (r *TypeRegistry) PrintType(typ *schemaspec.Type) (string, error) {
 			}
 			continue
 		}
-		lit, ok := arg.V.(*schemaspec.LiteralValue)
-		if !ok {
-			return "", errors.New("expecting literal value")
+		switch v := arg.V.(type) {
+		case *schemaspec.LiteralValue:
+			args = append(args, v.V)
+		case *schemaspec.ListValue:
+			for _, li := range v.V {
+				lit, ok := li.(*schemaspec.LiteralValue)
+				if !ok {
+					return "", errors.New("expecting literal value")
+				}
+				uq, err := strconv.Unquote(lit.V)
+				if err != nil {
+					return "", fmt.Errorf("expecting list items to be quoted strings: %w", err)
+				}
+
+				args = append(args, `'`+uq+`'`)
```

Comment:
```suggestion
				args = append(args, "'"+uq+"'")
```

Why backticks? 

I would leave it for the driver btw. Because there are cases where these strings need to be escaped ("a'b"), and replacing `"` with `'` is not always necessary.